### ISSUE TITLE
Incorrect metadata on SOS.NETCore.dll

### ISSUE THREAD

**Issue by morganbr** [dotnet/coreclr#11381]
SOS.NETCore.dll has incorrect assembly attributes and file metadata. This causes the .NET Native compiler to think it's not a framework assembly as well as looking wrong. The incorrect attributes I spotted are:
```
[assembly: AssemblyDefaultAlias("System.Private.CoreLib")]
[assembly: AssemblyDescription("System.Private.CoreLib")]
[assembly: AssemblyTitle("System.Private.CoreLib")]
```
It also is missing this attribute, which would fix the .NET Native problem:
```
[assembly: AssemblyMetadata(".NETFrameworkAssembly", "")]
```

The file metadata has File description "System.Private.CoreLib"

**Comment by mikem8361** [comment]
@morganbr msbuild is complaining about the dot "." in the above AssemblyMetadata attribute.

C:\ssd\coreclr2\src\ToolBox\SOS\NETCore\SOS.NETCore.csproj(39,62): error MSB4025: The project file could not be loaded. Name cannot begin with the '.' character, hexadecimal value 0x2E. Line 39,
position 62.

**Comment by mikem8361** [comment]
Never mind.  I was using the wrong msbuild AssemblyInfoLines syntax to add this attribute.

**Comment by mikem8361** [comment]
It looks like the other attributes have already been fixed in 2.0. PR #11395 adds the AssemblyMetadata attribute.

**Comment by mikem8361** [comment]
Fixed with PR #11395
